jd4/daemon.py

```diff
@@ -101,12 +103,13 @@ def __init__(self, session, request, ws):
         total_memory_usage_bytes = 0
         for index, case in enumerate(cases):
             status, score, time_usage_ns, memory_usage_bytes, stderr = await case.judge(sandbox, package)
-            self.next(case={'status': status,
+            self.next(status=STATUS_JUDGING,
+                      case={'status': status,
                             'score': score,
                             'time_ms': time_usage_ns // 1000000,
                             'memory_kb': memory_usage_bytes // 1024,
                             'judge_text': stderr.decode(encoding='utf-8', errors='replace')},
-                      progress=(index + 1) / len(cases))
+                      progress=(index + 1) / len(cases) * 100)
```

Comment:
This is already fixed as (index + 1) * 100 // len(cases)

We can make the progress integer after deprecating jd3